test/spec-wasm2c-prefix.c

```diff
@@ -202,59 +202,90 @@ static bool is_arithmetic_nan_f64(u64 x) {
   return (x & 0x7ff8000000000000) == 0x7ff8000000000000;
 }
 
+typedef struct Z_spectest_module_instance_t {
+  wasm_rt_table_t spectest_table;
+  wasm_rt_memory_t spectest_memory;
+  uint32_t spectest_global_i32;
+  uint64_t spectest_global_i64;
+  float spectest_global_f32;
+  double spectest_global_f64;
+} Z_spectest_module_instance_t;
+
+static Z_spectest_module_instance_t Z_spectest_module_instance;
 
 /*
  * spectest implementations
  */
-static void spectest_print(void) {
+void Z_spectestZ_print(Z_spectest_module_instance_t* module_instance) {
   printf("spectest.print()\n");
 }
 
-static void spectest_print_i32(uint32_t i) {
+void Z_spectestZ_print_i32(Z_spectest_module_instance_t* module_instance,
```

Comment:
I wonder if we should shorten this to `Z_spectest_instance_t* instance`?    i.e. is `modules_instance` needlessly verbose?

Reply:
(I think it might make this code more readable with less verbosity in these names .. but I could be overthinking).

Reply:
Agreed -- just pushed a rename from "module_instance" to "instance" (everywhere except the member of wasm_rt_funcref_t, where it seems useful to clarify that this is an instance of the originating module and not some other kind of instance).